listenbrainz/webserver/views/playlist_api.py

```diff
@@ -152,6 +153,113 @@ def serialize_jspf(playlist: Playlist):
     return {"playlist": pl}
 
 
+def serialize_xspf(playlist: Playlist):
+    """
+        Given a playlist, return a stringified element tree
+    """
+
+    playlist_root = ET.Element("playlist")
+    playlist_root.attrib = {
+        "version": "1",
+        "xmlns": "http://xspf.org/ns/0"
```

Comment:
The namespace declaration is missing a trailing / -- the https://validator.xspf.org/ validator complains about it not being there. It should be:

xmlns="http://xspf.org/ns/0/"